CenterForOpenScience/osf.io
# [#PLAT-384]Extract contact@osf.io and make them configurable

## Purpose
Extract contact@osf.io, contact@cos.io and make them configurable


## Changes
All contact@osf.io and contact@cos.io are now united to be contact@osf.io throughout osf and made configurable in website/settings. On the frontend, user shouldn't face any change except contact@cos.io has been changed to contact@osf.io.


## QA Notes
*Part 1. Check contact@osf.io function on OSF*
1.Go to staging.osf.io/support/ and confirm the link for contact@osf.io shows up correctly and, when clicked, actually mails to contact@osf.io
2.Repeat function check at https://staging.osf.io/12345/
3.Attempt in a few different browsers (IE, Edge, Safari, Firefox, Chrome), at tester discretion

*Part 2. Check contact@osf.io function in email to contributor_added*
1.Sign in to the OSF, create a project and add another of your registered OSF account B to that project as a contributor
2.The email address associated with the OSF account B will receive a verification email. Confirm the correct appearance and function of the contact@osf.io appears correctly at the bottom of  the email

*Part 3. Check contact@osf.io function in OSF welcome email*
1.Create a new osf account and then check the email and use the link in the email to confirm that account
2.You will receive a welcome email.  Confirm the correct appearance and function of the contact@osf.io link in the email


## Side Effects



## Ticket
https://openscience.atlassian.net/browse/PLAT-384
